cmd/provider/main.go

```diff
@@ -144,6 +148,23 @@ func main() {
 
 	log := logging.NewLogrLogger(zl.WithName("provider-ceph"))
 
+	// Init otel tracer provider
+	tp, err := traces.InitTracerProvider(*metricsExportAddress, *metricsExportTimeout, *metricsExportInterval)
+	if err != nil {
+		log.Debug("error starting tracer provider", "error", err.Error())
```

Comment:
Does it make sense to exit in this case?

Reply:
Yes, more so now that I added a flag to specifically enable tracing. Added.